packages/webcert/src/components/modal/CareProviderModal.test.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import { Provider } from 'react-redux'
+import { BrowserRouter } from 'react-router-dom'
+import store from '../../store/store'
+import CareProviderModal from './CareProviderModal'
+import { updateUser } from '../../store/user/userActions'
+import { LoginMethod, SigningMethod, Unit, User } from '@frontend/common'
+
+describe('Care provider modal', () => {
```

Comment:
More tests should be added either in this component or the child components. For example test that the units are printed, that the show/hide units functionality works etc.